ambry-api/src/main/java/com.github.ambry/config/CloudConfig.java

```diff
@@ -14,7 +14,38 @@
 package com.github.ambry.config;
 
 /**
- * The class for cloud related configurations.
+ * The configs for cloud related configurations.
  */
 public class CloudConfig {
+
+  @Config("vcr.cluster.zk.connect.string")
+  @Default("")
+  public final String vcrClusterZkConnectString;
+
+  /**
+   * The name of the associated vcr cluster for this node.
+   */
+  @Config("vcr.cluster.name")
```

Comment:
Please make the property names static constants, so the tests can refer to them without copying.